DTS-STN/dts-rmp
# 167 create modal template

# Description

Added Modal component that uses a slot (which we would pass in a form).

## Test Instructions

1. Navigate to 'Search contacts and engagements' select 'Engagements'. (All of this should be the Default/Home page)
2. Click on one of the engagement's button. The modal should pop up.
3. To show that different engagements will be passed, I pass the ID. Notice the last text line in the modal 

## Help Requested


## Checklist
